tooling/typescript/internal-package.json

```diff
@@ -5,6 +5,7 @@
     /** Emit types for internal packages to speed up editor performance. */
     "declaration": true,
     "declarationMap": true,
+    "emitDeclarationOnly": true,
```

Comment:
I think you may need to set noEmit to null here to properly override it. 

Otherwise we end up with the following error
error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.